framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -564,9 +566,32 @@ protected void addUncheckedCodeDefaults(QualifierDefaults defs) {
                         AnnotationUtils.fromClass(elements, annotation), TypeUseLocation.OTHERWISE);
             }
         }
+    }
+
+    /**
+     * Adds standard unchecked defaults that do not conflict with previously added defaults.
+     *
+     * @param defs {@link QualifierDefaults} object to which defaults are added
+     */
+    protected void addUncheckedStandardDefaults(QualifierDefaults defs) {
         Set<? extends AnnotationMirror> tops = this.qualHierarchy.getTopAnnotations();
         Set<? extends AnnotationMirror> bottoms = this.qualHierarchy.getBottomAnnotations();
         defs.addUncheckedStandardDefaults(tops, bottoms);
+    }
+
+    /**
+     * Ensure that we have defaults specified.
+     *
+     * @param defs {@link QualifierDefaults} object to which defaults are added
+     */
+    protected void ensureDefaultsExist(QualifierDefaults defs) {
```

Comment:
Yes, I made this a separate method so I could override it in checker-framework-inference where we might not necessarily have a default qualifier which is fine. Before we were adding Unqualified to our supported type qualifiers and thus sidestepped that error.